### PR TITLE
UISAUTCOMP-9: Add the marc view component, hide it after a filter change, add the ability to close it without redirection

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
## Purpose
see PR of [UIPFAUTH-5](https://github.com/folio-org/ui-plugin-find-authority/pull/18)

### CODE REVIEW

File: lib/SearchResultsList/SearchResultsList.js [hunk 1]
```diff
@@ -84,17 +72,11 @@ const SearchResultsList = ({
   hidePageIndices,
   hasNextPage,
   hasPrevPage,
+  onRenderHeadingRef,
```
Comment (BogdanDenis) [inline lib/SearchResultsList/SearchResultsList.js:75]: Better to name it `renderHeadingRef` - `on` prefix is used for callback functions

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: lib/AuthoritiesSearchPane/AuthoritiesSearchPane.js [hunk 2]
```diff
@@ -65,18 +69,23 @@ const AuthoritiesSearchPane = ({
...
         resetSelectedRows={resetSelectedRows}
         hasAdvancedSearch={hasAdvancedSearch}
       />
       {
         navigationSegmentValue === navigationSegments.browse
           ? (
-            <BrowseFilters cqlQuery={query} />
+            <BrowseFilters
+              cqlQuery={query}
+              onCloseDetailView={onCloseDetailView}
```
[the remark this answers is not in the document]
Comment (Dmytro-Melnyshyn) [reply lib/AuthoritiesSearchPane/AuthoritiesSearchPane.js:81]: Maybe we can just look at the filters in `AuthoritiesLookup`, what do you think?
```
const { navigationSegmentValue, filters } = useContext(AuthoritiesSearchContext);
useEffect(() => {
  closeDetailView();
}, [filters]);
```

**Review by BogdanDenis — APPROVED** [review]

File: lib/AuthoritiesSearchForm/AuthoritiesSearchForm.js [hunk 3]
```diff
@@ -213,6 +216,8 @@ AuthoritiesSearchForm.propTypes = propTypes;
 
 AuthoritiesSearchForm.defaultProps = {
   hasAdvancedSearch: false,
+  onChangeSortOption: null,
```
[the remark this answers is not in the document]
Comment (Dmytro-Melnyshyn) [reply lib/AuthoritiesSearchForm/AuthoritiesSearchForm.js]: Thanks, fixed

File: lib/SearchResultsList/SearchResultsList.js (hunk 1 above)
Comment (Dmytro-Melnyshyn) [reply lib/SearchResultsList/SearchResultsList.js]: Thanks, fixed

**Review by odoniyor — APPROVED** [review]

File: lib/AuthoritiesSearchPane/AuthoritiesSearchPane.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (BogdanDenis) [reply lib/AuthoritiesSearchPane/AuthoritiesSearchPane.js]: Sorry, I forgot about this comment and approved the PR
But just checking changes to filters won't work when searching by a different query, right?

[the remark this answers is not in the document]
Comment (Dmytro-Melnyshyn) [reply lib/AuthoritiesSearchPane/AuthoritiesSearchPane.js]: If the filters don't change, the `closeDetailView` function doesn't fire. It works for all search queries.

**Pull request merged**